go/cmd/dolt/dolt.go

```diff
@@ -120,6 +118,67 @@ var doltSubCommands = []cli.Command{
 	stashcmds.StashCommands,
 	&commands.Assist{},
 }
+
+var subCommandsUsingDEnv = []cli.Command{
```

Comment:
Should these be flags on a function or field in the cli.Command struct? Seems a little heavy weight.

Is your plan here to remove commands until there are no more?

Reply:
A flag or an empty interface would be a good approach here. Having a source of truth so far from the actual code can be problematic.